### PR TITLE
Fix for suckless.org

### PR DESCRIPTION
**Pull request by AndrewDaGuy**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -8276,6 +8276,22 @@ body {
 
 ================================
 
+suckless.org
+*.suckless.org
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8280]: This is redundant.

**Comment by AndrewDaGuy** [comment]
Why do i keep accidentally deleting the same branch over and over

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -8276,6 +8276,21 @@ body {
 
 ================================
 
+*.suckless.org
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:8279]: Change it to the right place and I meant the *.suckless.org is redunant.

Comment (AndrewDaGuy) [reply src/config/dynamic-theme-fixes.config]: 50fb573 Done

**Pull request merged**